NuGet-Src.slnf

```diff
@@ -0,0 +1,47 @@
+{
+  "solution": {
```

Comment:
Do we need your custom solution load? Is it optimized for all of us?

Reply:
Solution filters are meant to be adaptive depending on the scenario of the user. 
I created a SRC filter as that's been a useful sln filter I use. 

https://docs.microsoft.com/en-us/visualstudio/ide/filtered-solutions?view=vs-2019

You are not required to use it, so as such it needs to represent a common scenario.

Reply:
Any pattern to filter only unit test projects? I want to have only unit tests in solution, so that I can "Run All" without picking up all Apex/functional tests 🙂

Reply:
We don't label our unit test projects, so I think it'd have to be done manually. 
I think that's also a great example of a filter people would like :D 

Personally I also have a dotnet CLI & dotnet CLI + tests filters, but didn't want to push my luck!
Wanted to see how the team feels about it first.